lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -16,14 +16,14 @@
 from lms.djangoapps.certificates.data import CertificateStatuses
 from lms.djangoapps.courseware.context_processor import get_user_timezone_or_last_seen_timezone_or_utc
 from lms.djangoapps.course_goals.models import CourseGoal, CourseGoalReminderStatus, UserActivity
-from lms.djangoapps.course_goals.toggles import COURSE_GOALS_NUMBER_OF_DAYS_GOALS
 from openedx.core.djangoapps.ace_common.template_context import get_base_template_context
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 from openedx.core.djangoapps.user_api.preferences.api import get_user_preference
 from openedx.core.lib.celery.task_utils import emulate_http_request
 from openedx.features.course_duration_limits.access import get_user_course_expiration_date
+from openedx.features.course_experience import ENABLE_COURSE_GOALS
```

Comment:
This now feels odd to not be in the `course_goals` djangoapp, but it's fine to keep it where it is.

Is the medium term plan to remove this flag too, once we finish testing/rolling-out the goals feature? I see that you said Kaitlin recommended keeping one of the flags. But like, is this a permanent flag, designed to be a feature toggle? We might want to update its annotation comment and make it a waffle switch not a waffle flag then. Or is it to just let some courses opt out? Do we want to make this flag an opt-out toggle vs the current opt-in toggle? etc etc.

Just asking because right now, its annotations and usage feels like the kind of flag we use for rolling out, but we seem basically done with the rollout.

FYI, there is a very verbose OEP about feature toggles: https://github.com/openedx/open-edx-proposals/blob/master/oeps/best-practices/oep-0017-bp-feature-toggles.rst

Reply:
I don't mean to blow up your scope with this suggestion - you can ignore and it we can keep using this as a waffleflag if it's annoying to change it around. But documentation/annotation changes seem fine to squeeze in, if appropriate.

Reply:
My understanding from my conversation with @Ahernkb is that the idea was to see if there were concerns with the feature, and then if not, we could go ahead and remove the flag altogether.